Dockerfile

```diff
@@ -28,17 +28,16 @@ RUN dd bs=1024 count=2880 if=/dev/zero of=esp.img && \
       mcopy -i esp.img -v /tmp/grubx64.efi ::EFI/BOOT && \
       mdir -i esp.img ::EFI/BOOT
 
+FROM docker.io/centos:centos8
 
-FROM docker.io/centos:centos7
-
-RUN yum install -y python-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b train current-tripleo && \
-    yum update -y && \
-    yum install -y python-gunicorn openstack-ironic-api openstack-ironic-conductor crudini \
-        iproute dnsmasq httpd qemu-img-ev iscsi-initiator-utils parted gdisk psmisc \
-        sysvinit-tools mariadb-server genisoimage python-ironic-prometheus-exporter \
-        python-jinja2 && \
-    yum clean all && \
+RUN dnf install -y python3 python3-requests && \
```

Comment:
Installing python3-requests is supposed to pull in python 3, I wonder why it's not the case..

Reply:
this is "fun", I was also expecting that but apparently python3-requests doesn't care about python3!
The only dependencies are python3-chardet, python3-idna, python3-pysocks, python3-urllib3